src/datetime/enum/AmPm.php

```diff
@@ -0,0 +1,16 @@
+<?hh
+/*
+ *  Copyright (c) 2004-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+namespace HH\Lib\Experimental\DateTime;
+
+enum AmPm: string {
+  AM = 'am';
+  PM = 'pm';
+}
```

Comment:
should this really be a thing?

Reply:
Having a conversion from 24-hour time to 12-hour time is probably valuable, since it's not as simple as doing `($hours % 12)`, there are some edge cases around 12am and 12pm that people could forget about.

Whether this enum is valuable, I'm not sure. What would you propose as alternative? Return 'am' or 'pm' as string?

Reply:
yes, having a format for AM/PM instead so people can do $dt->format(...);

Reply:
`$dt->format('%I%P')` should work but I think it's still valuable to have the various getters, `format` shouldn't be the *only* way to access something

Reply:
This is a format tho! it's not a way to store things, if you really need PM/AM, you are most likely displaying it, in which case you should use format()